src/main/java/org/osc/controller/nsc/api/NeutronSdnControllerApi.java

```diff
@@ -39,41 +63,91 @@
                 SUPPORT_FAILURE_POLICY + ":Boolean=false",
                 USE_PROVIDER_CREDS + ":Boolean=true",
                 QUERY_PORT_INFO + ":Boolean=false",
-                SUPPORT_PORT_GROUP + ":Boolean=false" })
+                SUPPORT_PORT_GROUP + ":Boolean=false"},
+        service={SdnControllerApi.class},
```

Comment:
why is this needed now? also the immediate=true, IsmApplianceManagerApi also uses a db but does not have this.